### PR TITLE
Prepare release v2.1.0

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 can you please review ?

File: deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml [hunk 1]
```diff
@@ -100,8 +100,8 @@ spec:
             privileged: true
             capabilities:
               add: ["SYS_ADMIN"]
-          # for stable functionality replace canary with latest release version
-          image: quay.io/cephcsi/cephcsi:canary
+          # for stable functionality replace v2.1.0 with latest release version
```
Comment (Madhu-1) [inline deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml:103]: no need to update this comment, just keep it as it is. this applies to other places also

File: Makefile [hunk 2]
```diff
@@ -17,7 +17,7 @@
 CONTAINER_CMD?=docker
 
 CSI_IMAGE_NAME=$(if $(ENV_CSI_IMAGE_NAME),$(ENV_CSI_IMAGE_NAME),quay.io/cephcsi/cephcsi)
-CSI_IMAGE_VERSION=$(if $(ENV_CSI_IMAGE_VERSION),$(ENV_CSI_IMAGE_VERSION),canary)
+CSI_IMAGE_VERSION=$(if $(ENV_CSI_IMAGE_VERSION),$(ENV_CSI_IMAGE_VERSION), v2.1.0)
```
Comment (Madhu-1) [inline Makefile:20]:
```suggestion
CSI_IMAGE_VERSION=$(if $(ENV_CSI_IMAGE_VERSION),$(ENV_CSI_IMAGE_VERSION),v2.1.0)
```

File: deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml (hunk 1 above)
Comment (humblec) [reply deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml]: done.

**Comment by Madhu-1** [comment]
we will merge this PR once we backport all PR's to release-v2.1 branch

File: charts/ceph-csi-cephfs/Chart.yaml [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
 apiVersion: v1
-appVersion: canary
+appVersion: v2.1.0
 description: "Container Storage Interface (CSI) driver,
 provisioner, and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 1.3.0-canary
+version: 2.1.0-canary
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/Chart.yaml:7]: why the mix of version v2.1.0 and 2.1.0-canary? Can it not all be the same version?

I would also expect v2.1-canary, not v2.1.0-canary...

Comment (Madhu-1) [reply charts/ceph-csi-cephfs/Chart.yaml:7]: @nixpanic deploy.sh will replace will updates canary when pushing charts https://github.com/ceph/ceph-csi/blob/master/deploy.sh#L10

Comment (nixpanic) [reply charts/ceph-csi-cephfs/Chart.yaml:7]: In that case, should -canary not be used everywhere? Seems like version v2.1-canary can be used then.

File: .travis.yml [hunk 4]
```diff
@@ -16,6 +16,7 @@ language: go
 branches:
   only:
     - master
+    - release-v2.1
```
Comment (nixpanic) [inline .travis.yml:19]: this might fail to rebase? it was included in #944 too

Comment (Madhu-1) [reply .travis.yml:19]: Yes we don't need this anymore

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal

File: charts/ceph-csi-rbd/values.yaml [hunk 5]
```diff
@@ -87,7 +87,7 @@ nodeplugin:
   plugin:
     image:
       repository: quay.io/cephcsi/cephcsi
-      tag: canary
+      tag: 2.1.0
```
Comment (Madhu-1) [inline charts/ceph-csi-rbd/values.yaml:90]: @humblec this need to be v2.1.0. that why CI is failing

Comment (Madhu-1) [reply charts/ceph-csi-rbd/values.yaml:90]: awesome, we would have made same issue as https://github.com/ceph/ceph-csi/issues/829. as we have E2E for helm we got this issue

Comment (humblec) [reply charts/ceph-csi-rbd/values.yaml]: done.. thanks @Madhu-1

**Comment by Madhu-1** [comment]
Merging this one manually, as this is not authored by the bot

**Pull request merged**